tests/unittest/_torch/multi_gpu/test_moe_a2a_rank_mask.py

```
@@ -0,0 +1,407 @@
+    [
+        (4, 16, 2),
+        (4, 32, 4),
+    ],
+    indirect=["mpi_pool_executor"],
+)
+def test_all_active_mask_matches_no_mask(mpi_pool_executor, local_num_tokens, top_k):
+    """An all-ones active_rank_mask must produce identical output to omitting it."""
+    try:
+        MnnvlMemory.initialize()
+        assert MnnvlMemory.supports_mnnvl()
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)